master/platformio.ini

```diff
@@ -4,15 +4,17 @@ default_envs = teensy41, teensy41-debug
 [env:teensy41]
 platform = teensy
 build_flags = -D EMBEDDED
-board = teensy41
+;board = teensy41
+board = teensy40
```

Comment:
why the change here, shouldn't it be another environment?

Reply:
yes i pushed the wrong thing, for testing we need to use teensy40 as that's the breadboard testing board